src/Components/ContainerItem/ContainerItem.js

```diff
@@ -307,6 +313,129 @@ export default function ContainerItem({
+														<Heading
+															align="center"
+															weight="normal"
+															size={6}
+															style={{
+																display: 'flex',
+																alignItems: 'center',
+																justifyContent: 'space-between',
+																margin: 0,
+															}}
+														>
+															<span style={{ position: 'relative' }}>
+																Filter
+																<div className="tooltip_block">
+																	<img
+																		className="input_icon"
+																		src="/img/icons/info_circle.svg"
+																		height={18}
+																		width={18}
+																		alt="info"
+																	/>
+																	<div className="tooltip">Flexible search allows you to use any attributes to search through the attribute-operation-value formula, you can use parameters such as FilePath, FileName, etc.</div>
+																</div>
+															</span>
+															<Button
+																color="primary"
+																size="small"
+																onClick={() => {
+																	let filtersTemp = [ ...filters ];
+																	filtersTemp.push({
+																		key: '',
+																		match: 'MatchStringEqual',
+																		value: '',
+																	});
+																	setFilters(filtersTemp);
+																}}
+																disabled={isLoadingObjects}
+															>
+																Add filter
+															</Button>
+														</Heading>
+														{filters.map((filterItem, indexFilter) => (
+															<div className="filter_item" key={indexFilter}>
+																<Form.Control>
+																	<Form.Input
+																		size='small'
+																		placeholder="Attribute"
+																		value={filterItem.key}
+																		onChange={(e) => {
+																			let filtersTemp = [ ...filters ];
+																			filtersTemp[indexFilter].key = e.target.value;
+																			setFilters(filtersTemp);
+																		}}
+																		disabled={isLoadingObjects}
+																	/>
+																</Form.Control>
+																<Form.Control className="select_wrapper">
+																	<Form.Select
+																		size='small'
+																		value={filterItem.match}
+																		onChange={(e) => {
+																			let filtersTemp = [ ...filters ];
+																			filtersTemp[indexFilter].match = e.target.value;
+																			setFilters(filtersTemp);
+																		}}
+																		disabled={isLoadingObjects}
+																	>
+																		{[
+																			{ value: 'MatchStringEqual', title: '=' },
+																			{ value: 'MatchStringNotEqual', title: '≠' },
+																			{ value: 'MatchNumGT', title: '>' },
+																			{ value: 'MatchNumGE', title: '≥' },
+																			{ value: 'MatchNumLT', title: '<' },
+																			{ value: 'MatchNumLE', title: '≤' },
+																		].map((item) => (
```

Comment:
As I understand that the rest supports only 4 of these [parameters](https://github.com/nspcc-dev/neofs-rest-gw/blob/master/spec/rest.yaml#L1764-L1767), and all the parameters are kind of listed [here](https://github.com/nspcc-dev/neofs-sdk-go/blob/a14b155381748007f55586a0f2d97b0c62737d41/eacl/enums.go#L75-L98).
 
What are we supposed to do with other matches (>=,...)?

<img width="666" alt="Screenshot 2024-12-05 at 13 03 29" src="https://github.com/user-attachments/assets/ce0731c0-f203-44a2-9db1-a9e328604a8d">

Now it's not working